src/utils/components/DiskListTitle/DiskListTitle.tsx

```diff
@@ -11,14 +11,15 @@ const DiskListTitle = () => {
 
   return (
     <Title headingLevel="h2">
-      <SearchItem id="disks">{t('Disks')}</SearchItem>
-      <HelpTextIcon
-        bodyContent={t(
-          'The following information is provided by the OpenShift Virtualization operator.',
-        )}
-        helpIconClassName="title-help-text-icon"
```

Comment:
Now I think this line was here to prevent being the "?" too big in the title, as it is now...